cmd/ocm-load-test.go

```diff
@@ -125,6 +125,30 @@ func run(cmd *cobra.Command, args []string) error {
 		viper.Set("tests.all", map[string]interface{}{})
 	}
 
+	// Require CCS Credentials if `create-cluster` test is going to be be ran
+	/* WORK IN PROGRESS
+	isCCSRequired := false
+	if viper.Get("tests.all") != nil {
+		isCCSRequired = true
+	}
+	for _, testName := range viper.GetStringSlice("test-names") {
+		if testName == "create-cluster" {
+			isCCSRequired = true
+		}
+	}
+	if isCCSRequired {
+		if len(viper.GetString("ccsAccountID")) == 0 {
+			log.Fatal("ccsAccountID is required to run the create-cluster test.")
+		}
+		if len(viper.GetString("ccsAccessKey")) == 0 {
+			log.Fatal("ccsAccessKey is required to run the create-cluster test.")
+		}
+		if len(viper.GetString("ccsSecretKey")) == 0 {
+			log.Fatal("ccsSecretKey is required to run the create-cluster test.")
+		}
+	}
+	*/
```

Comment:
@kurtismullins why is this commented out?

Reply:
That was temporary while the PR was a draft.